openedx/core/djangoapps/bookmarks/models.py

```diff
@@ -81,7 +81,7 @@ def create(cls, data):
 
             xblock_cache = XBlockCache.create({
                 'usage_key': usage_key,
-                'display_name': block.display_name,
+                'display_name': block.display_name or '',
```

Comment:
Should there be some sort of default name when there isn't a display name? For instance, the method display_name_with_default could be used. Or the model definition could use a canned name.

How does the UI look with the empty string? I imagine it is confusing.